pyro/contrib/gp/models/svgp.py

```diff
@@ -0,0 +1,146 @@
+from __future__ import absolute_import, division, print_function
+
+import torch
+from torch.autograd import Variable
+from torch.distributions import constraints, transform_to
+from torch.nn import Parameter
+
+import pyro
+import pyro.distributions as dist
+from pyro.distributions.util import matrix_triangular_solve_compat
+
+from .model import Model
+
+
+class SparseVariationalGP(Model):
+    """
+    Sparse Variational Gaussian Process module.
+
+    References
+
+    [1] `Scalable variational Gaussian process classification`,
+    James Hensman, Alexander G. de G. Matthews, Zoubin Ghahramani
+
+    :param torch.autograd.Variable X: A tensor of inputs.
```

Comment:
there needs to be more info about the dimensions of things.... it seems you only allow the input to be 1 dimensional?

Reply:
Yes, I will add some info there. Right now, the input is 2D (though 1D is also fine). However, the output is just a 1D tensor. Supporting multidimensional output will be addressed this in a future pull request.